public/app/features/explore/utils/ResultProcessor.ts

```diff
@@ -128,8 +127,25 @@ export function isTimeSeries(frame: DataFrame, datasource?: string): boolean {
   return false;
 }
 
+/* In Explore table, we don't want to show time-series.
```

Comment:
This is not true , we want to show time series in table!!!  But we can’t because then it would show that instead of the instant results for Prometheus . But for ALL other datasources we want to show the table data for no matter what

Reply:
Yeah, my comment wasn't clear. I meant  **In Explore table, we don't want to show single-value time series (series with exactly 2 fields - time and value)**. But based on your comment, that is not true as well.

Currently (on master), we show in table only time series with 3+ fields. So as you can see [here](https://play.grafana.org/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22InfluxDB%22,%7B%22datasource%22:%22InfluxDB%22,%22policy%22:%22default%22,%22resultFormat%22:%22time_series%22,%22orderByTime%22:%22ASC%22,%22tags%22:%5B%5D,%22groupBy%22:%5B%7B%22type%22:%22time%22,%22params%22:%5B%22$__interval%22%5D%7D,%7B%22type%22:%22fill%22,%22params%22:%5B%22null%22%5D%7D%5D,%22select%22:%5B%5B%7B%22type%22:%22field%22,%22params%22:%5B%22value%22%5D%7D,%7B%22type%22:%22mean%22,%22params%22:%5B%5D%7D%5D%5D,%22measurement%22:%22logins.count%22%7D,%7B%22mode%22:%22Metrics%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D), Explore table doesn't have any series in table. But we would like to see something like this, right?:

![image](https://user-images.githubusercontent.com/30407135/80528235-e696db00-8995-11ea-96d5-536a87bd7782.png)

So maybe the staring point should be clarifying what we want/do't want to show. Based on my observation and your feedback:
- Prometheus - in table, we want to show only results of instant queries.
- Elastic - in table, if we have data frames with time series with 3+ fields, show that in table. If we don't, should we show single-value time series (which we use for graph)?
- Influx and rest of the data sources - in table, we want to show all time series.